types/src/front/lib/connectors_api.ts

```diff
@@ -390,6 +390,15 @@ export class ConnectorsAPI {
   async getConnector(
     connectorId: string
   ): Promise<ConnectorsAPIResponse<ConnectorType>> {
+    const parsedId = parseInt(connectorId, 10);
```

Comment:
Just to confirm, it's supposed to always be a number here? Looking at the prototype it's confusing (`connectorId: string`).

Reply:
Looks like it yes. `getConnector` is retrieving the connector this way:
```
const connector = await ConnectorResource.fetchById(req.params.connector_id);
```

Reply:
I agree very confusing

Reply:
Make sure to monitor. You might want to log as well.